docs/user_guide/configuring.rst

```diff
@@ -19,20 +19,78 @@ doc path's _static folder, and use the following configuration:
 
    html_logo = "_static/logo.png"
 
-Configure social media buttons
-==============================
+Configure icon links
+====================
 
-If you'd like social media buttons to show up to the right of your nav bar, use the
+If you'd like icon links to show up to the right of your main navigation bar, use the
 following configuration:
 
 .. code:: python
 
    html_theme_options = {
-     "github_url": "https://github.com/<your-org>/<your-repo>",
-     "gitlab_url": "https://gitlab.com/<your-org>/<your-repo>",
-     "twitter_url": "https://twitter.com/<your-handle>",
+       ...
+       "icon_links": [
+           {
+               "name": "GitHub",
+               "url": "https://github.com/<your-org>/<your-repo>",
+               "icon": "fab fa-github-square",
+           },
+           {
+               "name": "GitLab",
+               "url": "https://gitlab.com/<your-org>/<your-repo>",
+               "icon": "fab fa-gitlab",
+           },
+           {
+               "name": "Twitter",
+               "url": "https://twitter.com/<your-handle>",
+               "icon": "fab fa-twitter-square",
+           },
+       ],
+       ...
    }
 
+
+The value of ``icon`` can be any full
+`FontAwesome 5 Free <https://fontawesome.com/icons?d=gallery&m=free>`__.
```

Comment:
Seems to be missing a word in the sentence.

Reply:
fixed in d616236d!